app/views/users/profile.html.erb

```diff
@@ -249,13 +249,14 @@
 
   <div class="info float:left">
     <ul>
-    <li class="mt-3"><h5>@<%= @profile_user.name %> has</h5></li> 
+    <li class="mt-3"><h5>@<%= @profile_user.name %> has</h5></li>
       <li><h5><a href='/notes/author/<%= @profile_user.name %>'><%= @profile_user.note_count %> research notes</a></h5></li>
       <li><h5><%= @profile_user.revisions.count %> wiki edits</h5></li>
       <li><h5><a href = "/tag/question:*/author/<%= params[:id] %>"><%= Node.questions.where(status: 1, uid: @profile_user.id).length %> questions </a></h5></li>
       <li><h5><%= Comment.where(uid: @profile_user.id).count %> comments</h5></li>
       <li><h5><a href = "/tag/activity:*/author/<%= params[:id] %>"><%= @count_activities_posted %> activities posted </a></h5></kli>
       <li><h5><a href = "/tag/replication:*/author/<%= params[:id] %>"><%= @count_activities_attempted %> activities attempted</a> </h5></li>
+      <li><h5><a href = "/tag/with:<%= @profile_user.username %>"><%= Tag.find_nodes_by_type("with:#{@profile_user.id}").count %> notes co-authored</a> </h5></li>
```

Comment:
Lets change this to 
```
<li><h5><a href = "/tag/with:<%= @profile_user.username %>"><%= Tag.find_nodes_by_type("with:#{@profile_user.username}").count %> notes co-authored</a> </h5></li>

```
And we should be ready to merge thanks for the patience @WulfPlasma